sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample9_DifferentiateOutputModelsTrainedWithAndWithoutLabels.cs

```diff
@@ -0,0 +1,118 @@
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.AI.FormRecognizer.Training;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        /// This sample demonstrates the differences in output that arise when StartRecognizeCustomForms
+        /// is called with custom models trained with labels and without labels.
+
+        [Test]
+        public async Task OutputModelsTrainedWithLabels()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            string trainingFileUrl = TestEnvironment.BlobContainerSasUrl;
+            string formFilePath = FormRecognizerTestEnvironment.CreatePath("Form_1.jpg");
+
+            FormTrainingClient trainingClient = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Model trained with labels
+            CustomFormModel modelWithLabels = await trainingClient.StartTraining(new Uri(trainingFileUrl), useTrainingLabels: true).WaitForCompletionAsync();
+
+            using (FileStream stream = new FileStream(formFilePath, FileMode.Open))
+            {
+                RecognizedFormCollection formsWithLabels = await client.StartRecognizeCustomForms(modelWithLabels.ModelId, stream).WaitForCompletionAsync();
+
+                // With a form recognized by a model trained with labels, the 'field.Name' key will be its
+                // training-time label. Note that Label data is not returned for model trained with labels.
+                Console.WriteLine("---------Recognizing forms with models trained with labels---------");
+                foreach (RecognizedForm form in formsWithLabels)
+                {
+                    Console.WriteLine($"Form of type: {form.FormType}");
+                    foreach (FormField field in form.Fields.Values)
+                    {
+                        Console.WriteLine($"Field {field.Name}: ");
+                        Console.WriteLine($"    Value: '{field.ValueText.Text}");
+                        Console.WriteLine($"    Confidence: '{field.Confidence}");
+                    }
+                }
+
+                // Find a specific labeled field.
+                // For this particular sample, we will look for the known training-time label 'VendorName'.
+                Console.WriteLine("Find the value for a specific labeled field:");
+                foreach (RecognizedForm form in formsWithLabels)
+                {
+                    if (form.Fields.TryGetValue("VendorName", out FormField field))
```

Comment:
Because we are searching for a specific field in the Form_1.jpg would it make sense to provide the training set in the samples folder? As I was writing this sample, I realized that we tell users to train a model, but we don't provide them with the training files.

Reply:
Hmm interesting... I'm not sure. Do you know if the service has/exposes these files as samples? I will prefer pointed to something like that...

Reply:
Not sure that they do. You could always point to the python repo if you'd like.

Reply:
Could you show me where in Python? I see the [documents](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_forms/forms) but not the training ones.

Reply:
It's still in PR, I have a few more changes I want to make: https://github.com/Azure/azure-sdk-for-python/pull/12164

Reply:
hmm what do you think about putting the files in a public/read blob so all repos can point to the same files and user can go and download them?

Reply:
updated

Reply:
😮 we can do that? Is there any security concerns?

Reply:
Discussed internally with the team.
We can :) so I went and created a container with all the files we need and has public read access.
https://aka.ms/azsdk/formrecognizer/docs/trainingdocs

I updated my PR with it

Reply:
That's awesome, thanks so much for checking on that and creating the container!

Reply:
I think we should also add that link to the training samples

Reply:
Sorry for all the comments but I'm getting `(2011) Could not access Azure blob storage account` when I use that link. Does it work for you?

Reply:
>I think we should also add that link to the training samples

Will add

Reply:
hmm weird.
Yeah I did a quick call and was able to see the files. 
```
string endpoint = "https://mariaritrainingpublic.blob.core.windows.net/training";
var container = new BlobContainerClient(new Uri(endpoint));

Console.WriteLine(container.GetBlobs().Count());
```
I'm using Azure.Storage.Blobs Version="12.4.4"

Reply:
It works if I specify `endpoint` like you do above. If I pass in the aka.ms link it does not work. Trying to figure out why

Reply:
Hmmm the redirection doesn't work though...
I wonder then, if we should only have the link and not the aka.ms

Reply:
Should we ask Ted? He seemed to have it working for the storage sample.

Reply:
asking...

Reply:
Redirect is not properly working on the service side so we decided to add the files in each repo. 
Will update this PR with that